assets/src/domain/shared/entities/datetimes/predicates/filters/activeUpcoming/index.ts

```diff
@@ -1,10 +1,9 @@
-/**
- * Internal dependencies
- */
+import isActive from '../../isActive';
+import isUpcoming from '../../isUpcoming';
 import { DatetimeFilterFn } from '../types';
 
 const activeUpcoming: DatetimeFilterFn = (dates) => {
-	return dates.filter(({ isActive, isUpcoming }) => isActive || isUpcoming);
+	return dates.filter((date) => isActive(date) || isUpcoming(date));
```

Comment:
You could do this
```js
R.filter(R.anyPass([ isActive, isUpcoming ]), dates);
```

Reply:
As discussed in chat I was getting some different results with `anyPass`. that's why I've decided to go simpler here
![image](https://user-images.githubusercontent.com/1477453/75049024-28a52b80-54d2-11ea-87dc-09e625d30b04.png)